domains/vela.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "v3laDev",
+        "email": "27khero@proton.me"
+    },
+    "record": {
+        "RECORDTYPE": "CNAME"
```

Comment:
Your RECORDTYPE should be an actual record and the value should be the value you want to place for that record.

See docs: https://docs.is-a.dev

Reply:
alr